Leaderboard/secret.js

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs';
+import config from 'config';
+import crypto from 'crypto';
+// TODO: Is this the best way to handle application secrets?
```

Comment:
When an application first starts up, it will securely generate a 512 byte secret and store it in a file. I'm open to doing it other ways, but this way we don't have to have any user interaction when setting up the application, and it always behaves in a secure manner. Error handling probably isn't optimal, but I believe that if there is an error, it won't allow the application to start up.